DataFormats/TrackerRecHit2D/src/classes_def.xml

```diff
@@ -131,17 +131,17 @@
 
   // fastsim containers
   
-  <class name="FastTrackerRecHit" ClassVersion="10">
-    <version ClassVersion="10" checksum="1775447631"/>
+  <class name="FastTrackerRecHit" ClassVersion="11">
+    <version ClassVersion="11" checksum="1771133479"/>
```

Comment:
old versions should be kept

Reply:
When I reverted to the old version, I get an error on compile:

Checking EDM Class Version for src/DataFormats/TrackerRecHit2D/src/classes_def.xml in libDataFormatsTrackerRecHit2D.so
error: class 'FastTrackerRecHit' has a different checksum for ClassVersion 10. Increment ClassVersion to 11 and assign it to checksum 1771133479

Reply:
You need to keep both, i.e.
```xml
  <class name="FastTrackerRecHit" ClassVersion="11">
    <version ClassVersion="10" checksum="1775447631"/>
    <version ClassVersion="11" checksum="1771133479"/>
  </class>
```

Reply:
right, that's what I meant

Reply:
Ah, ok